### DISCUSSION TITLE
keyboard usage:   Deleting entire row(s) ; selecting entire row(s) ; navigate to table/DB tree pane

### DISCUSSION THREAD

**Discussion by shula** [pawelsalawa/sqlitestudio#4977] [General]
I'm often deleting entire rows (via GUI).   But I can't find an easy way with keyboard shortcuts.

1. Is there a shortcut to select entire row(s)  -- ?
In this i keep shift pressed, and press right-arrow until it reaches the end of the table. And only then the entire row is selected (indicated by the highlighted row-numbers).
I couldn't find in the settings.

2. Feature Request: 
a key combo to erase entire row.

currently, to delete an entire row(s) I have to 
 A) click the DB navigation pane; 
 B) expand the db, 
 C) expand the Tables, 
 D) doube click the table (only works with mouse-clicks!)
 E) select "Data" tab
 F) select a cell (or multiple cells, with shiftup/down)
 G) shift + RIGHT arrow, till the last column
 H) press DEL
 I) press commit (Ctrl+Enter)

3. is there a way to show Table Data only with the keyboard?
4. Ideally, is there a way to open a table data-view directly from command line?
  e.g.    C:\> sqliteStudio.exe file.sqlite --table=items --view=data   

thank you

**Comment by pawelsalawa** [comment]
There is no such option at the moment. There is already similar request reported - #67
